R/steps.R

```diff
@@ -110,7 +110,9 @@ Cicerone <- R6::R6Class(
 
       assertthat::assert_that(tabs_ok(tab, tab_id))
 
-      if(is_id)
+      # If a class
+      is_id <- !grepl("^\\.", el)
```

Comment:
We no longer need `is_id` if we infer from `el`

Reply:
Ok, I can remove that. I'll search the package and make sure it's not used anywhere else.

Reply:
Done in 25667d2